roles/mongodb-common/tasks/main.yml

```diff
@@ -33,6 +33,15 @@
     - ursula_os == 'rhel'
     - openstack_install_method == 'distro'
 
+- name: Create keyFile
+  copy:
+    dest: "{{ mongodb.keyFile }}"
+    content: "{{ mongodb.mongodb_keyfile_content }}"
```

Comment:
probably should remove redundant mongodb from variable name. i.e rename mongodb_keyfile_content to keyfile_content